src/learning/environment.yml

```diff
@@ -0,0 +1,83 @@
+name: cycleganlearning
+channels:
+  - defaults
+dependencies:
+  - backcall=0.1.0
+  - blas=1.0
+  - ca-certificates=2019.1.23
+  - certifi=2018.11.29
+  - cffi=1.11.5
+  - cudatoolkit=9.0
+  - cudnn=7.3.1
+  - cycler=0.10.0
+  - dbus=1.13.6
+  - decorator=4.3.2
+  - expat=2.2.6
+  - fontconfig=2.13.0
+  - freetype=2.9.1
+  - glib=2.56.2
+  - gst-plugins-base=1.14.0
+  - gstreamer=1.14.0
+  - icu=58.2
+  - intel-openmp=2019.1
+  - ipython=7.2.0
+  - ipython_genutils=0.2.0
+  - jedi=0.13.2
+  - jpeg=9b
+  - kiwisolver=1.0.1
+  - libedit=3.1.20181209
+  - libffi=3.2.1
+  - libgcc-ng=8.2.0
+  - libgfortran-ng=7.3.0
+  - libpng=1.6.36
+  - libstdcxx-ng=8.2.0
+  - libtiff=4.0.10
+  - libuuid=1.0.3
+  - libxcb=1.13
+  - libxml2=2.9.9
+  - matplotlib=3.0.2
+  - mkl=2018.0.3
+  - mkl_fft=1.0.6
+  - mkl_random=1.0.1
+  - nccl=1.3.5
+  - ncurses=6.1
+  - ninja=1.8.2
+  - numpy=1.15.4
+  - numpy-base=1.15.4
+  - olefile=0.46
+  - openssl=1.1.1a
+  - parso=0.3.2
+  - pcre=8.42
+  - pexpect=4.6.0
+  - pickleshare=0.7.5
+  - pillow=5.4.1
+  - pip=19.0.1
+  - prompt_toolkit=2.0.8
+  - ptyprocess=0.6.0
+  - pycparser=2.19
+  - pygments=2.3.1
+  - pyparsing=2.3.1
+  - pyqt=5.9.2
+  - python=3.6.8
+  - python-dateutil=2.7.5
+  - pytorch=0.4.1
+  - pytz=2018.9
+  - qt=5.9.7
+  - readline=7.0
+  - setuptools=40.8.0
+  - sip=4.19.8
+  - six=1.12.0
+  - sqlite=3.26.0
+  - tk=8.6.8
+  - torchvision=0.2.1
+  - tornado=5.1.1
+  - traitlets=4.3.2
+  - wcwidth=0.1.7
+  - wheel=0.32.3
+  - xz=5.2.4
+  - zlib=1.2.11
+  - zstd=1.3.7
+  - pip:
+    - torch==0.4.1
+prefix: /home/kevin/.conda/envs/cycleganlearning
```

Comment:
It seems like the prefix might not be needed when building the environment: https://github.com/conda/conda/issues/4339.
```suggestion

```

Reply:
This file should not exist in the first place.
Deleted.